### PR TITLE
Bugfix: LIVE-3847 dex swap quote add bsc

### PR DESCRIPTION
**Pull request by sarneijim**
### 📝 Description

BSC has been already implemented but there is an error (it is comparing family, while family for bsc is ethereum)

### ❓ Context

- **Impacted projects**: LLD
- **Linked resource(s)**:  https://ledgerhq.atlassian.net/browse/LIVE-3847

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Review by chabroA — APPROVED** [review]

File: .changeset/fluffy-beans-lie.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"ledger-live-desktop": major
```
Comment (chabroA) [inline .changeset/fluffy-beans-lie.md:2]: Should not be a major but rather a minor

```suggestion
"ledger-live-desktop": minor
```

**Review by chabroA — CHANGES_REQUESTED** [review]

File: .changeset/fluffy-beans-lie.md (hunk 1 above)
Comment (sarneijim) [reply .changeset/fluffy-beans-lie.md]: Thanks!

**Review by chabroA — APPROVED** [review]

**Comment by ggilchrist-ledger** [comment]
The tokens are appearing and the quotes are good, but ETH is selected in the DEX when we navigate to it. Is that expected and outside the scope of this ticket? @sarneijim

**Comment by sarneijim** [comment]
> The tokens are appearing and the quotes are good, but ETH is selected in the DEX when we navigate to it. Is that expected and outside the scope of this ticket? @sarneijim
> 
>  Screen.Recording.2023-01-11.at.14.10.59.mov

Yes, this is the dex swap deep-link scope.

**Pull request merged**